04 Abstraction Exercises Part 1/credit-card-application/src/main/java/CreditCard.java

```diff
@@ -0,0 +1,63 @@
+class CreditCard {
+    private static double creditLimit;
+    private String accountNumber;
+    private String ownerName;
+    private String cvv;
+    private String expirationDate;
```

Comment:
why is expirationDate string?

Reply:
Para po flexible yung expirationdate sa kahit anong date format